File: eBarberShop/UI/ebarbershop_mobile/lib/models/ocjene/ocjene_insert_request.dart
import 'package:json_annotation/json_annotation.dart';

part 'ocjene_insert_request.g.dart';

@JsonSerializable()
class OcjeneInsertRequest {
  DateTime datum;
  double ocjena;
  String? opis;
  int uposlenikId;
  int korisnikId;

  OcjeneInsertRequest(
      this.datum, this.ocjena, this.opis, this.uposlenikId, this.korisnikId);

  factory OcjeneInsertRequest.fromJson(Map<String, dynamic> json) =>
      _$OcjeneInsertRequestFromJson(json);

  Map<String, dynamic> toJson() => _$OcjeneInsertRequestToJson(this);
}

File: eBarberShop/eBarberShop.Model/Requests/OcjeneInsertRequest.cs
namespace eBarberShop.Model.Requests
{
    public class OcjeneInsertRequest
    {
        public DateTime Datum { get; set; }
        public double Ocjena { get; set; }
        public string? Opis { get; set; }
        public int UposlenikId { get; set; }
        public int KorisnikId { get; set; }
    }
}
